Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM php:8.0.0rc1-cli-alpine
+
+WORKDIR /usr/local/bauhaus
+
+COPY docker/* /opt/php-libs/files/
```

Comment:
could be only `COPY ./.docker /` and the folder structure could be defined in the project